doc/configuration.rst

```diff
@@ -654,6 +654,9 @@ point to the directory containing ``searchindex.js``, such as
 If you wish to do the same for ordinary reST documentation,
 see :ref:`plain_rst`.
 
+The parameter ``embed_code_links`` can be used to disable links in code cells
+but keep them in text cells.
+
```

Comment:
I would actually move this up higher, so people can find out about this config quicker, e.g.:

```
By default Sphinx-Gallery adds hyperlinks in your example scripts so that
you can link used functions/methods/attributes/objects/classes to their matching
online documentation. To disable this for code blocks, but keep them for text
blocks, set the configuration ``embed_code_links`` to ``False``::

    sphinx_gallery_conf = {
        ...
        'embed_code_links': False
    }

``embed_code_links`` is ``True`` by default, which causes code snippets within
the gallery will appear like this:
```